core/trino-main/src/main/java/io/trino/sql/planner/optimizations/BeginTableWrite.java

```diff
@@ -22,10 +22,12 @@
 import io.trino.sql.planner.PlanNodeIdAllocator;
```

Comment:
Just a minor observation: there's no such a thing as "UPDATE SET". The feature is called "UPDATE".